DDR4 OC Guide.md

```diff
@@ -198,8 +198,9 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
   | :-:        | :-: | :--: | :-: | :--: |
   | 8Gb AFR    | Y   | N    | N   | ?    | 
   | 8Gb CJR    | Y   | N    | N   | Y    |
-  | 8Gb Rev. E | Y   | N    | Y   | ?    |
+  | 8Gb Rev. E | Y   | N    | N   | N    |
   | 8Gb B-die  | Y   | Y    | Y   | Y    |
+  * tCWL and tRDWR are linked, one of them can be tightened together with tCL by the same amount if voltage is increased.
```

Comment:
Can you clarify what you mean by
> one of them can be tightened together with tCL by the same amount

Reply:
for example, tCL 14, tCWL 12 and tRDWR 10 at 1.45V
if bumping voltage to 1.55V allows tCL to go down to 13, either tCWL or tRDWR can go down by 1
resulting in, tCL 13, tCWL 12 and tRDWR 9 or tCWL 11 and tRDWR 10

tCWL and tRDWR are linked, this works if tWRRD is manually set, preferably to 1, at least on amd

Reply:
![1](https://user-images.githubusercontent.com/16512539/102746429-61dc4f80-43b2-11eb-8534-a47a3189f804.PNG)

Going up to 1.50v allows tCL 14 but doesn't allow tRDWR 12, at least on my Nanya sticks.

Reply:
Fair enough, I thought tCL and tCWL scale with voltage on all chips and by extension so would tRDWR but it looks like I was wrong. So it's not universal, perhaps it is isolated to a number of chips or even only rev e.

Reply:
```suggestion
```
Great. Can you remove that line so that we can merge?